test/tests/functional/pbs_equiv_classes.py

```diff
@@ -1495,29 +1495,30 @@ def test_server_queue_limit(self):
         # Set queue limit
         a = {
             'max_run': '[o:PBS_ALL=100],[g:PBS_GENERIC=20],\
-                       [u:PBS_GENERIC=20],[g:tstgrp01 = 8],[u:pbsuser1=10]'}
+                       [u:PBS_GENERIC=20],[g:tstgrp01 = 8],[u:%s=10]' %
```

Comment:
Since you are fixing usernames, you should fix the group names while you are at it.

Reply:
I think it would be reasonable to file a bug to fix group names.

Reply:
Since it won't take long to fix the group names, I'd rather do it now.